backend/prisma/schema.prisma

```diff
@@ -58,4 +58,13 @@ model Lecture {
   chapter   Chapter  @relation(fields: [chapterId], references: [id])
   userId    Int    
   user      User     @relation(fields: [userId], references: [id])
+  sections  Section[]
 }
+
+model Section {
+  id        Int       @id @default(autoincrement())
+  name      String
+  photo     String?   // You can adjust the type based on your needs
```

Comment:
Not a big deal, but this chatgpt comment is not useful to us, so I would remove it